docs-ref-services/data-lake-store.md

```diff
@@ -18,17 +18,37 @@ ms.service: Data Lake Store
 ## Overview

```

Comment:
Someone went back through and lowercased all of our Modules to match the sentence case specification of the style guide. I have updated this gist to reflect this. https://gist.github.com/burkeholland/5262966fc3070dccc7c184163c9f19ec